src/Entity/CaptchaPoint.php

```diff
@@ -94,6 +99,14 @@ public function getCaptchaType() {
   /**
    * {@inheritdoc}
    */
+  public function disable() {
+    Cache::invalidateTags($this->getCacheTags());
+    return $this->setStatus(FALSE);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
   public function setCaptchaType($captcha_type) {
     $this->captchaType = $captcha_type != 'default' ? $captcha_type : NULL;
   }
```

Comment:
Adding the disable/enable form and implementing the disable method completely (with a cache invalidation).